examples/llm/main.py

```diff
@@ -26,6 +28,40 @@ def build_composer_model(cfg):
         raise ValueError(f'Not sure how to build model with name={cfg.name}')
 
 
+def get_evaluators_from_config(cfg):
+    tokenizer = TOKENIZER_REGISTRY[cfg.tokenizer.type](**cfg.tokenizer.args)
+    evaluators = []
+    for eval_cfg in cfg.icl_tasks:
+        dataset_uri = eval_cfg.get('dataset_uri')
```

Comment:
same comment about the "gets" here. Assuming these are all required, we don't want to silently have them be `None` and then die later.